content/blog/2020/10/2020-10-31-spring-xstream.adoc

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Spring and XStream updates"
```

Comment:
Wondering whether we should have some slightly more catchy title so users in average read it more.
Like "Spring and XStream updates: you won't believe the incredible things we had to do!" :trollface:

Reply:
If you have a _serious_ suggestion please let me know. I agree we want this to grab attention.

Reply:
I suppose something like "Spring and XStream updates: potential impacts on Jenkins installations" would catch a bit more eyes?